packages/grid/src/vaadin-grid-mixin.js

```diff
@@ -0,0 +1,1025 @@
+/**
+ * @license
+ * Copyright (c) 2016 - 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import './vaadin-grid-styles.js';
```

Comment:
We probably could keep grid styles outside the mixin to apply them using `static get styles()` in the Lit version.  Not a blocker, current approach also works.

Reply:
Moved to `vaadin-grid.js`